ford/utils.py

```diff
@@ -24,9 +24,17 @@
 #  
 
 
-
 import re
 import os.path
+import json
+import sys
+import ford.sourceform
+if (sys.version_info[0] > 2):
+    from urllib.request import urlopen
+    from urllib.parse import urljoin
+else:
+    from urllib import urlopen
+    from urlparse import urljoin
```

Comment:
Don't worry about supporting Python 2, it's dead :)

```suggestion
from urllib.request import urlopen
from urllib.parse import urljoin
```

Reply:
Well, the change is several years old, and back then we still cared about Python 2 ;) It's fine to do away with it.